sdk/communication/communication-sms/src/generated/src/smsApiClientContext.ts

```diff
@@ -10,7 +10,7 @@ import * as coreHttp from "@azure/core-http";
 import { SmsApiClientOptionalParams } from "./models";
 
 const packageName = "azure-communication-sms";
-const packageVersion = "1.0.1";
+const packageVersion = "1.0.0";
```

Comment:
I'm not sure why this package version changed. It seems like it was incorrectly manually changed previously.